tests/std/tests/P0896R4_views_transform/test.cpp

```diff
@@ -784,4 +775,19 @@ int main() {
 
         (void) views::transform(Fn{})(span<int>{});
     }
+
+    { // Validate that iter_swap works when result of transformation is a lvalue reference

```

Comment:
Grammar corner case: This should say "an" because even though "lvalue" starts with a consonant, it's pronounced as the letter "L" followed by "value", and when spoken, "el" starts with a vowel sound.

```suggestion
    { // Validate that iter_swap works when result of transformation is an lvalue reference
```

Existing examples:

https://github.com/microsoft/STL/blob/d6f9987d7ec694b8599fe21e87e3653c36566223/stl/inc/type_traits#L254
https://github.com/microsoft/STL/blob/d6f9987d7ec694b8599fe21e87e3653c36566223/stl/inc/type_traits#L263

Other examples of letters with consonant and vowel sounds (here, the first letter controls "a" vs. "an"):

* "My old computer had a CRT." ("a see are tee")
* "My new computer has an SSD." ("an ess ess dee")
* "This Mars rover has an RTG." ("an are tee gee")